src/popup/components/TransferReview.vue

```diff
@@ -224,8 +224,9 @@ export default defineComponent({
             amount,
             recipient,
             hash: actionResult.hash,
-            type: 'spendToken',
             pendingTokenTx: true,
+            pending: true,
```

Comment:
`pending` field is being added by the `addPendingTransaction` action itselft

Reply:
`addPendingTransaction` is a store function, so there's no types there. I added it here as the `pending` property is required by the `IPendingTransaction`. In the future, when we will start to remove the Vuex we can try to do it better.